### DISCUSSION TITLE
Image area for project

### DISCUSSION THREAD

**Discussion by nbentoneves** [sproogen/modern-resume-theme#110] [Ideas]
**Is your feature request related to a problem? Please describe.**
Isn't related to any problem. It's a feature I think will be good for the project section. 

**Describe the solution you'd like**
Create the capability to add images in projects, like screenshots or a logo related to the project

**Describe alternatives you've considered**

**Additional context**
**Like this:**
![suggestion](https://user-images.githubusercontent.com/3682693/97273952-e2a42280-182b-11eb-977a-1083bd79a8c0.png)

**Comment by sproogen** [comment]
Thanks for this, I've just updating how content is managed but this is definitely something I will look at once I've finished this part.

**Reply by jihoray** [reply]
Hi, is there any way that I can add the images like above? This is very nice template by the way! Thanks a lot.

**Reply by FasihZ** [reply]
Hello, can you please share how you managed to add the image? Thank you!

**Comment by FasihZ** [comment]
Hello, can you please share how you managed to add the image? Thank you!
